.github/workflows/release.yml

```diff
@@ -0,0 +1,49 @@
+name: Create Zipapp and Release
+
+on:
+    push:
+        tags:
+            - "v*"
+
+jobs:
+    build:
+        runs-on: ubuntu-latest
+
+        steps:
+            - uses: actions/checkout@v2
+
+            - name: Set up Python 3.9
+              uses: actions/setup-python@v2
+              with:
+                  python-version: 3.9
+
+            - name: Install dependencies
+              run: |
+                  python -m pip install --upgrade pip
+                  pip install shiv
+
+            - name: Create Zipapp
+              run: shiv -o cloudsmith-cli.pyz -e cloudsmith_cli.cli.commands.main:main .
+
+            - name: Create Release
+              id: create_release
+              uses: actions/create-release@v1
+              env:
+                  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+              with:
+                  tag_name: ${{ github.ref }}
+                  release_name: Release ${{ github.ref }}
+                  body: ""
+                  draft: false
+                  prerelease: false
+
+            - name: Upload Release Asset
+              id: upload-release-asset
+              uses: actions/upload-release-asset@v1
+              env:
+                  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+              with:
+                  upload_url: ${{ steps.create_release.outputs.upload_url }}
+                  asset_path: ./cloudsmith-cli.pyz
+                  asset_name: cloudsmith-cli.pyz
```

Comment:
should we put the version number in the name? that's usually best practice, just makes it simpler for people downloading multiple versions

Reply:
When you `pip install` the CLI it uses the name `cloudsmith`, I think we should match that, rather than `cloudsmith-cli`